packages/hardhat/hardhat.config.ts

```diff
@@ -33,7 +34,7 @@ const config: HardhatUserConfig = {
       },
     ],
   },
-  defaultNetwork: "localhost",
+  defaultNetwork: "sepolia",
```

Comment:
I usually don't change this, because unwarned people might deploy to the incorrect network by mistake. When I want to deploy to another network I add the flag on the command:
```
yarn deploy --network sepolia
```

Reply:
[RESOLVED]